mirrord-operator/templates/namespace.yaml

```
@@ -1,4 +1,6 @@
+{{- if .Values.createNamespace }}
 apiVersion: v1
 kind: Namespace
 metadata:
   name: {{ .Values.namespace }}
+{{ end }}
```

Let's not remove the new line at the end of the file.
```suggestion
{{ end }}

```